test/parallel/test-http-outgoing-message-inheritance.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const common = require('../common');
+const { OutgoingMessage } = require('http');
+const { Writable } = require('stream');
+const assert = require('assert');
+
+// check that OutgoingMessage can be used without a proper Socket
```

Comment:
Can you link to the issues in the test files? I think that would be helpful for people looking at these tests at a later point.